OpenTaiko/Global/PuchiChara/02 - Bol/Metadata.json

```diff
@@ -1,5 +1,16 @@
 {
-	"name" : "Bol",
+	"name" : {
+		"strings": {
+			"default": "Bol",
+			"ru": "Бол",
+			"zh": "包子"
```

Comment:
Unconfirmed Chinese translation. Bol is a character for an OpTk charter named Bol.

Reply:
Recalled.